utils/rpms/daos.spec

```diff
@@ -38,8 +38,10 @@ BuildRequires: libpmem-devel >= 1.8, libpmemobj-devel >= 1.8
 BuildRequires: fuse3-devel >= 3.4.2
 %if (0%{?suse_version} >= 1500)
 BuildRequires: libprotobuf-c-devel
+BuildRequires: liblz4-devel
 %else
 BuildRequires: protobuf-c-devel
+BuildRequires: lz4-devel
```

Comment:
@brianjmurrell It sounds like the package has a different name on centos and leap. Did i insert this in the right place? I wasn't sure ...

Reply:
Seems good now at least.